### PR TITLE
add support for additional prometheus labels for query

### PR DESCRIPTION
**Pull request by harshavardhana**

### CODE REVIEW

File: .golangci.yml [hunk 1]
```diff
@@ -16,10 +16,17 @@ linters:
...
+    - gofmt
     - unused
     - structcheck
-    - goheader
+    - unconvert
+    - varcheck
+    - gocritic
+    - gofumpt
+    - tenv
+    - durationcheck
```
Comment (harshavardhana) [inline .golangci.yml:29]: Enabled these linters so had to fix all the issues fixed in this PR

**Comment by harshavardhana** [comment]
This PR adds support for additional labels for Prometheus query, extending from not just `job=job-id` to additional labels - refer here https://github.com/minio/minio/issues/14353

**Review by prakashsvmx — APPROVED** [review]
LGTM :+1:

**Review by prakashsvmx — APPROVED** [review]

**Review by bexsoft — APPROVED** [review]
LGTM

**Comment by harshavardhana** [comment]
PTAL @dvaldivia @bexsoft @Alevsk

**Review by prakashsvmx — APPROVED** [review]

**Review by prakashsvmx — APPROVED** [review]

**Comment by harshavardhana** [comment]
PTAL @bexsoft @dvaldivia

**Review by dvaldivia — APPROVED** [review]

**Pull request merged**